src/transformStateWithClones.js

```diff
@@ -7,7 +7,42 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let resultObj = { ...state };
```

Comment:
It is better not to use `result` everywhere in your variable's name until it's really result